app/views/guidance-for-farmers.njk

```diff
@@ -279,18 +283,38 @@
         You must make sure the review takes place and you claim funding within 6 months of receiving your agreement number.
       </p>
 
+      <p class="govuk-body">
+        If the review has not taken place and you need more time to complete the review, contact the RPA.
+      </p>
+      <p class="govuk-body">
+        If the review has taken place, and you have a good reason why you won’t be able to make your claim within 6 months - such as illness, contact the RPA as soon as you can.
+      </p>
+
       <h2 id="gethelp" class="govuk-heading-m">Get help with your application or claim</h2>
 
       <p class="govuk-body">
-        If you cannot apply or claim online or have a question about your application or claim, contact the Rural Payments Agency.
+        Contact the RPA if you:
       </p>
-      <p class="govuk-body">Have your agreement number with you when you call or include it in your email if you have it.</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>decide you do not want a review after you’ve accepted the agreement offer</li>
+        <li>cannot apply or claim online or have a question about your application or claim</li>
+      </ul>
+      
+      <p class="govuk-body">Have your agreement number with you when you call or include it in your email or letter.</p>
+      
       <ul class="govuk-list">
```

Comment:
this block looks like a common thing across multiple pages, shoudn't we change that everywhere?

Reply:
Content designers determine this.